app/javascript/components/contact.js

```diff
@@ -1,10 +1,19 @@
 
 const showMorePersonnalCollection = () => {
   const showMore = document.getElementById("button-show-personnal");
-  const hiddenExhibition =  document.querySelector("#hidden-personnal-exhibitions");
+  const hiddenExhibition =  document.querySelector("#showed-personnal-exhibitions");
   showMore.addEventListener('click',(e) => {
-    hiddenExhibition.classList.toggle('exhibitions-hidden');
+    hiddenExhibition.classList.toggle('swap');
   });
 };
 
```

Comment:
It's a really nice idea, but for now it doesn't work properly. 
The "button" doesn't always respond, when going back to initial position, the list stop sometimes at like 8 instances  before going back to 5. And sometimes, the thing goes on till there is no instance anymore.
I guess you are not quite finished ?

Reply:
Pain in the ass, behavior with bootstrap, and because i do 2 .each for each category. It's a big fix to do

Reply:
i added to github action

Reply:
Github actions : things to do, pain in the ass )